src/components/Sections/SectionProgress.tsx

```diff
@@ -1,41 +1,83 @@
 import React from 'react'
-import { Icon, Label, Progress } from 'semantic-ui-react'
-import {
-  ApplicationProgress,
-  ReviewAndConsolidationProgress,
-  ReviewProgress,
-  SectionState,
-} from '../../utils/types'
+import { Label, Progress } from 'semantic-ui-react'
+import { ApplicationProgress, ConsolidationProgress, ReviewProgress } from '../../utils/types'
 import strings from '../../utils/constants'
 
-const ReviewStatusOrProgress: React.FC<SectionState> = ({
-  reviewProgress,
-  reviewAction,
-  reviewAndConsolidationProgress,
-}) => {
-  if (reviewAction?.isAssignedToCurrentUser && reviewProgress) {
-    return reviewAction.isReviewable ? (
-      <ReviewSectionProgressBar
-        reviewProgress={reviewProgress}
-        reviewAndConsolidationProgress={reviewAndConsolidationProgress}
-      />
-    ) : (
-      <Label
-        icon={<Icon name="circle" size="mini" color="blue" />}
-        content={strings.LABEL_ASSIGNED_TO_YOU}
+interface SectionProgressBarProps {
+  consolidationProgress?: ConsolidationProgress
+  reviewProgress?: ReviewProgress
+}
+
+/**
+ * Application
+ */
+
+const getApplicationProgressTitle = ({ completed, valid }: ApplicationProgress) => {
+  if (!valid) return strings.LABEL_SECTION_PROBLEM
+  else if (completed) return strings.LABEL_SECTION_COMPLETED
+  return null
+}
+
+const ApplicationProgressBar: React.FC<ApplicationProgress> = (applicationProgress) => {
```

Comment:
would be good to keep this in the same place when making changes (easier for reviewer to see diff)

Reply:
Oopsy 😯